jaeger_client/reporter.py

```diff
@@ -120,14 +122,29 @@ def __init__(self, channel, queue_capacity=100, batch_size=10,
             self.flush_interval = flush_interval or None
             self.io_loop.spawn_callback(self._consume_queue)
 
-        self._process_lock = Lock()
-        self._process = None
+    @staticmethod
+    def _fetch_io_loop(channel, sender):
```

Comment:
Quick question: `_fetch_io_loop` is used once. What's the advantage to using a static method vs inlining?

Reply:
This was introduced in https://github.com/jaegertracing/jaeger-client-python/pull/186/commits/435b5329bb0fb605342c9dd4542064522d2d696c, and from the looks of it is was for readability and testability.  Considering the io loop should come from the sender, I went ahead and corrected inline.